src/compiler/separate_erasure_compiler.nit

```diff
@@ -39,6 +39,11 @@ redef class ToolContext
 		if opt_no_check_all.value then
 			opt_no_check_erasure_cast.value = true
 		end
+
+		# Temporally disabled. TODO: implement tagging in the erasure compiler.
```

Comment:
You certainly mean "temporary" unless this disabling evolves based on a time consideration.